.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+<!-- Thank you for sending a PR !-->
+
+Please perform the following checks and tick mark all the points.
+
+- [ ] The page (if new), does not already exist in the repo.
+
+- [ ] The page (if new), is added to the correct platform folder.
+i.e. If it's common to all platforms, it is under `common/` and if it's linux specific, it is under `linux/`. If the tool works on all unix-y platforms, please add multiple copies of the same page under all folders.
```

Comment:
This renders correctly as-is, but for extra neatness, I'd suggest indenting the line to align with the text of the line above, and wrapping it by inserting a line break after "\`linux/\`."

Here too I'd suggest changing "it is under" to some other wording, perhaps something simpler to piggyback on the first sentence:

    - [ ] The page (if new), is added to the correct platform folder,
          i.e. `common/` common to all platforms, `linux/` if it's Linux-specific, and so on.

I think the line about working the same on unix-y platforms is too specific -- we could leave such pages on `common/`, which IIRC is already the case for the majority of pages in that folder.

At most, we could add a line about tools that do exist in multiple platforms but with subtle differences (tipically linux vs mac), but that too seems overly specific and wordy for a generic PR template, IMO.